FXyz-Importers/src/main/java/org/fxyz3d/importers/Model.java

```diff
@@ -0,0 +1,55 @@
+package org.fxyz3d.importers;
```

Comment:
The license header is missing

Reply:
I'd use the FXyz license in this case, updated to 2019.

Reply:
I'll grab the LICENSE one then, with 2013-2019